R/DataDiscoveryRetrieval.R

```diff
@@ -23,7 +23,12 @@
 #' 
 #' @export
 #'
-
+#' @examples 
```

Comment:
I'd highly recommend adding more examples to all the functions. These will get tested as the package is checked. Also, they get run and the output is displayed in the pkgdown output. I'm a huge fan of lots of examples.

Reply:
Great suggestion, do you recommend to always include \donttest for examples like this?

Reply:
If you ever expect to push TADA to CRAN, you'll want to add `\donttest` to any example that makes an internet call. If your function is just doing something local, you don't need the donttest. Also, if you never plan to put it on CRAN then it's not a big deal either.